reconcile/queries.py

```diff
@@ -704,10 +707,26 @@ def get_app_interface_sql_queries():
 """
 
 
-def get_saas_files():
+def get_saas_files(saas_file_name='', env_name=''):
     """ Returns SaasFile resources defined in app-interface """
     gqlapi = gql.get_api()
-    return gqlapi.query(SAAS_FILES_QUERY)['saas_files']
+    saas_files = gqlapi.query(SAAS_FILES_QUERY)['saas_files']
+
+    for saas_file in saas_files[:]:
+        if saas_file_name:
```

Comment:
placing the if before the for would avoid processing the loop

Reply:
added a condition before the for loop